EIPS/eip-5023.md

```diff
@@ -0,0 +1,143 @@
+
+These requirements have set us to define shareable non-transferable NFTs. These shareable NFTs can be “shared” in the same way digital goods can be shared, at an almost zero technical transaction cost. They are used to instantiate quantified anti-rival value, a medium of sharing. Hence, they work somewhat as money, being a store of value and a unit of account, but instead of being a medium of exchange, they are a medium of sharing.
+
+Typical NFT standards such as ERC-721 and ERC-1155 do not define a sharing modality. Instead ERC standards define user interfaces for typical rival use cases such as token minting and token transactions that the NFT contract implementations should fulfil. The ‘standard’ contract implementations may extend the functionalities beyond the definition of interfaces. The tokens developed in the ATARCA experiments are designed to be token standard compatible at the interface level. However the implementation of token contracts may contain extended functionalities to match the requirements of the experiments such as the requirement of 'shareability'. In reflection to standard token definitions, shareability of a token could be thought of as re-mintability of an existing token. Contracts define re-mintable non-transferable tokens which retain some reference to previous tokens upon and after re-minting.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+```solidity
+interface IERCsntNFT is IERC165 {
+
+  /// @dev This emits when a token is shared, reminted and given to another wallet that isn't function caller
+  event Share(address indexed from, address indexed to, uint256 indexed tokenId, uint256 derivedFromtokenId);
+
+  /// @dev Shares, remints an existing token, gives a newly minted token a fresh token id, keeps original token at function callers possession and transfers newly minted token to receiver which should be another address than function caller. 
+  function share(address to, uint256 tokenIdToBeShared, uint256 newTokenId) external virtual;
+
+} 
+```
+
+## Rationale
+
+Current NFT standards define transferable non-fungible tokens. However to be able to create shareable NFTs we see that existing NFT contracts could be extended with an interface which defines the basic principles of sharing, namely the Event of sharing and the function method of sharing. In ATARCA we have chosen to go with shareable non-transferable NFTs which in our use case denote an award or merit for done effort. 
+
+Shareable tokens can be transferable or non-transferable as it is up to implementation details of the token contract. In the reference implementation we have a general case distilled from the ATARCA case that defines a shareable non-transferable NFTs using the shareable NFT interface.
+
+
```

Comment:
A quick blurb about why you don't have any of the standard token functions (like `ownerOf`, for example) would be useful. Do you expect implementers to also implement another standard alongside this one, like `ERC721` from EIP-721, for example?

Reply:
Excellent question @SamWilsn! A short answer would be yes, we expect that users would extend their ERC token implementations e.g. ERC-721 or ERC-1155 with this interface given that they want to make them shareable.

A lengthier answer is that we've been tackling the concept of shareability and shareable tokens for a while now and what we have noticed is that depending on the context shareability can take different forms, and one might use different types implementations for shareable tokens. Hence we haven't restricted that the interface must fulfil a specific token type. Though to add to that, in our use cases we have mostly been working with non-fungible token standards such as ERC-721 tokens, but we've also found some use for the ERC-1155 nft tokens. 

What I personally think is that the wider audience would greatly benefit from an abstraction level higher definition for shareability, such as this interface implementation, that defines minimum amount of functions that would be implemented to satisfy shareability. 

Let me know what you think. Any feedback is highly appreciated.

Reply:
I think that's good information, and could be added to the Rationale section.

Reply:
Excellent, I'll incorporate the gist to the rationale section.